sdks/java/extensions/sql/jdbc/build.gradle

```diff
@@ -0,0 +1,51 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * License); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+apply from: project(":").file("build_rules.gradle")
```

Comment:
Out of curiosity, also question for @lukecwik, is it necessary to have a separate module? Is it just easier because Gradle has a bunch of assumptions? One of the key selling points of Gradle was that it would be target/task based a la make/bazel/pants/etc so you wouldn't have this proliferation of config files for doing very little, and that you could have reusable build rules.

Reply:
Looking at the PR, its a toss up whether you want to place it inside the sql project and just specialize which tests run under which configuration with which classpath. You could use the same source set, or define new ones, or go with a separate module route.

Reply:
OK gotcha. Thanks for the details. It just adds lots of accidental complexity to the project directories and config files but we should do things the way Gradle expects to stay simple and canonical.